src/transformers/models/owlvit/feature_extraction_owlvit.py

```diff
@@ -50,13 +50,13 @@ class OwlViTFeatureExtractor(FeatureExtractionMixin, ImageFeatureExtractionMixin
     Args:
         do_resize (`bool`, *optional*, defaults to `True`):
             Whether to resize the shorter edge of the input to a certain `size`.
-        size (`int`, *optional*, defaults to 768):
+        size (`int`, *optional*, defaults to (768, 768):
             Resize the shorter edge of the input to the given size. Only has an effect if `do_resize` is set to `True`.
```

Comment:
Would be great to clarify here whether it should be (width, height) or (height, width).

Normally, it should be (width, height), as we're using Pillow for resizing, right?

Reply:
Yes, exactly! I will clarify it to avoid confusion though